### DISCUSSION TITLE
Получение порядкового номера недели МЕСЯЦА по заданной дате

### DISCUSSION THREAD

**Discussion by Vadimgrig2012** [totumonline/totum-ru-issues-and-discussions#548] [Как это сделать?] [answered]
Добрый день. Подскажите пожалуйста, как можно получить порядковый номер недели МЕСЯЦА по заданной дате?  Я в доке смотрю - можно стандартным функционалом получить только порядковый номер недели ГОДА по заданной дате... У меня всегда проблема с калькуляцией дат)

**Comment by erpic-ru** [comment, marked as answer]
Как-то так:

```
week: $date1 - $date3 + 1

date1: dateFormat(date: $#nd; format: "W")
date2: dateFormat(date: $#nd; format: "Y-m-01") 
date3: dateFormat(date: $date2; format: "W")
```

**Reply by Vadimgrig2012** [reply]
Спасибо за помощь!
